config/locales/js-en.yml

```diff
@@ -1043,13 +1043,17 @@ en:
         createdAt: "Created on"
         description: "Description"
         date: "Date"
+        done_ratio: "% Complete"
+        done_ratio_alternative: "Progress"
```

Comment:
As you named the other 2 fields after their new name, can you also name `done_ratio[_alternative]` as `percent_complete[_alternative]`?

Reply:
Also I'm not sure if mixing cases is ok there. Shouldn't it be snakeCase like others?

Reply:
I saw snake case and camel case being used alternatively so didn't think we enforce a style. I do know camel is the commonly accepted one so I'll do so :)

Reply:
Eayh, I'm not familiar with frontend so I suppose camelCase is the way to go there.